interfaces/builtin/gnome_online_accounts_service.go

```diff
@@ -0,0 +1,77 @@
+// -*- Mode: Go; indent-tabs-mode: t -*-
+
+/*
+ * Copyright (C) 2017 Canonical Ltd
+ *
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License version 3 as
+ * published by the Free Software Foundation.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program.  If not, see <http://www.gnu.org/licenses/>.
+ *
+ */
+
+package builtin
+
+const gnomeOnlineAccountsServiceSummary = `allows operating as the GNOME Online Accounts service`
+
+const gnomeOnlineAccountsServiceBaseDeclarationSlots = `
+  gnome-online-accounts-service:
+    allow-installation:
+      slot-snap-type:
+        - core
+    deny-connection: true
+`
+
+const gnomeOnlineAccountsServiceConnectedPlugAppArmor = `
+# Description: Allow access to GNOME Online Accounts service
+
+#include <abstractions/dbus-session-strict>
+
+# Allow use of ObjectManager APIs, used to enumerate accounts
+dbus (receive, send)
+    bus=session
+    interface=org.freedesktop.DBus.ObjectManager
+    path=/org/gnome/OnlineAccounts
+    peer=(label=unconfined),
```

Comment:
This is only true on classic. On core this would be confined. If this interface is only for classic then I'd suggest renaming the summary to say something like `allows interacting with the GNOME Online Accounts service.`

Reply:
FYI, this is only on classic (`implicitOnCore` is not set). The summary of the language looks fine AFAICT in this context.